### DISCUSSION TITLE
Config File Help

### DISCUSSION THREAD

**Discussion by ChadChaddington12345** [Flexget/Flexget#4027] [Q&A]
Good afternoon, newbie in most ways but essentially got my own media server running and trying to have automated downloads for certain parameters I have set, like 1080p only, etc. I am running into issues with flexget and the config file. I am not sure what I am doing wrong and if someone has the time to perhaps help me set this up that would be GREATLY appreciated. Running newest Mint distro, I used 2 spaces when starting a new line or more accordingly from reading the flexget guide. I replaced the URL I was using for RSS because it gave the following error:
```
Malformed configuration file (check messages above). Common reasons:

-------------------------------------------------------------------------------

o Indentation error

o Missing : from end of the line

o Non ASCII characters (use UTF8)

o If text contains any of :[]{}% characters it must be single-quoted (eg. value{1} should be 'value{1}')

Reason: mapping values are not allowed here

----> MOST LIKELY REASON: Missing : from end of the line!

Check configuration near line 3, column 10

Fault is almost always in this or previous line
```
Here is what my configuration file looks like, I replaced the url with the xml thinking that may work but then got this error:
```
[/tasks/test-task/rss] '/home/user/.config/flexget/283877.rss.xml/' is not valid under any of the given schemas

[/tasks/test-task/rss] '/home/user/.config/flexget/283877.rss.xml/' is not valid under any of the given schemas
```
```yaml
tasks:
  test-task:
    rss: /home/bruno/.config/flexget/283877.rss.xml/
    series:
      - Show:
          quality: 1080p
      - Show:
          quality: 1080p
      - Show:
          quality: 1080p
    download: /media/user/TOSHIBA EXT1/Television
```
Basically I just want to download a handful of shows and movies (really only movies from one provider, not sure how to figure that out) and have it be set up to do so the next day after airing so will need help setting up those variables as well. My experience level is not that great but I have an eagerness to learn and find this all fascinating and want to take it as far as I am able. Thank you!!

**Comment by ChadChaddington12345** [comment]
I noticed it removed the spacing and things from here when I posted it.
